### ISSUE TITLE
Firebug Script tab

### ISSUE THREAD

**Issue by Wancieho** [angular/angular-cli#1417]
> Please provide us with the following information:
> ---------------------------------------------------------------
1. OS?  Windows 7, 8 or 10. Linux (which distribution). Mac OSX (Yosemite? El Capitan?)
Windows 10

2. Versions. Please run `ng --version`. If there's nothing outputted, please
angular-cli: 1.0.0-beta.10
node: 5.8.0
os: win32 x64

3. Repro steps. Was this an app that wasn't created using the CLI? What change did you
   do on your code? etc.
Yes, nothing changed

4. The log given by the failure. Normally this include a stack trace and some
   more information.
Error: @http://localhost:4200/system-config.js:34:5
	Call@http://localhost:4200/vendor/es6-shim/es6-shim.js:289:14
	forEach@http://localhost:4200/vendor/es6-shim/es6-shim.js:1295:14
	@http://localhost:4200/system-config.js:33:1
	Zone</ZoneDelegate</ZoneDelegate.prototype.invoke@http://localhost:4200/vendor/zone.js/dist/zone.js:323:20
	Zone</Zone</Zone.prototype.run@http://localhost:4200/vendor/zone.js/dist/zone.js:216:25
	scheduleResolveOrReject/<@http://localhost:4200/vendor/zone.js/dist/zone.js:571:53
	Zone</ZoneDelegate</ZoneDelegate.prototype.invokeTask@http://localhost:4200/vendor/zone.js/dist/zone.js:356:24
	Zone</Zone</Zone.prototype.runTask@http://localhost:4200/vendor/zone.js/dist/zone.js:256:29
	drainMicroTaskQueue@http://localhost:4200/vendor/zone.js/dist/zone.js:474:26
	ZoneTask/this.invoke@http://localhost:4200/vendor/zone.js/dist/zone.js:426:22
	
	Evaluating http://localhost:4200/system-config.js
	Error loading http://localhost:4200/system-config.js
http://localhost:4200/vendor/systemjs/dist/system.src.js
Line 121

5. Mention any other details that might be useful.
In the scripts tab it stops at this file: http://localhost:4200/vendor/reflect-metadata/Reflect.ts
The error disappears if I disable the Firebug Script tab.

**Comment by jalbr74** [comment]
I'm seeing this exact same problem.  
`ng --version` yields the following:

> angular-cli: 1.0.0-beta.10
node: 6.2.2
os: win32 x64

I see the error in my FireBug console after creating a new project using `ng new`, running `ng serve`, and then opening `http://localhost:4200/` in a FireFox browser with FireBug enabled.

**Comment by czinkos** [comment]
Similar issue here.

$ ng --version
angular-cli: 1.0.0-beta.10
node: 5.10.1
os: win32 ia32

1. 'ng new' works
2. 'ng serve' works
3. modifying app.component works
4. 'ng g service my-service' works
5. import my-service in app.component/adding to contructor for injection) does not work
6. remove service import from app.component - does not work, neither

Same systemjs related output in firebug console.

**Comment by filipesilva** [comment]
Can you verify this still happens in the upcoming `webpack` release? The build system changed drastically in #1455 and I don't know if this is still a problem.

**Comment by jaykhimani** [comment]
Facing the same issue. Tried it on windows 7 and os x both. 

```
npm - 3.10.5
node - 6.3.1
```
```
ng --version
(node:11616) fs: re-evaluating native module sources is not supported. If you are using the graceful-fs module, please update it to a more recent version.
angular-cli: 1.0.0-beta.10
node: 6.3.1
os: win32 x64
```
```
1. ng new my-app
2. cd my-app
3. ng serve
4. http://localhost:4200 shows the app find with text - app works!
5. Stop the app. (ctrl + c)
6. ng serve (without any changes to the project)
7. http://localhost:4200 shows Loading... and browser console shows the error as mentioned [by @Wancieho](https://github.com/angular/angular-cli/issues/1417#issue-167036833)
```

@filipesilva how to verify the ```webpack``` thing? FYI, am quite new to whole angular/node thing. So excuse me if this being a stupid question :-)

**Comment by jaykhimani** [comment]
Ah between I just noticed that this happens only on Firefox (hence the title Firebug Script tab?). For Chrome it works. Also for IE 11.

**Comment by czinkos** [comment]
Hi, I couldn't reproduce the error with the webpack branch. Thanks.

$ ng -v
angular-cli: local (v1.0.0-beta.10, branch: webpack)
node: 5.7.0
os: linux x64

**Comment by maroshi** [comment]
Reproduced the problem on Ubunto 16.04
node: 4.4.7
angular-cli: 1.0.0-beta.10
os: linux x64
Firefox: 47.0

Chromium and Google Chrome works fine.

**Comment by maroshi** [comment]
Reproduced the problem on Ubunto 16.04
node: 4.4.7
angular-cli: 1.0.0-beta.10
os: linux x64
Firefox: 47.0

Chromium and Google Chrome works fine.
Apparently this is a conflict with FireBug. Restarting Firefox in safemode works.

**Comment by filipesilva** [comment]
Fixed by #1455. Problem no longer exists on the webpack branch as reported by @czinkos.
